src/base/system.cpp

```diff
@@ -3666,13 +3666,26 @@ int open_link(const char *link)
 	MultiByteToWideChar(CP_UTF8, 0, link, -1, wBuffer, sizeof(wBuffer) / sizeof(WCHAR));
 	return (uintptr_t)ShellExecuteW(NULL, L"open", wBuffer, NULL, NULL, SW_SHOWDEFAULT) > 32;
 #elif defined(CONF_PLATFORM_LINUX)
-	char aBuf[512];
-	str_format(aBuf, sizeof(aBuf), "xdg-open %s >/dev/null 2>&1 &", link);
-	return system(aBuf) == 0;
+	const int pid = fork();
+	if(pid == 0)
+		execl("/usr/bin/env", "env", "xdg-open", link, nullptr);
```

Comment:
```suggestion
		execl("xdg-open", link, nullptr);
```
Double `env`. None required, I think.

Reply:
Done without env. The double open is required though since the first argument has to be the name of the program.